release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -3016,6 +3016,50 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.18.10
+[id="ocp-4-18-10_{context}"]
+=== RHSA-2025:4019 - {product-title} {product-version}.10 bug fix update and security update
+
+Issued: 22 April 2025
+
+{product-title} release {product-version}.10 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:4019[RHSA-2025:4019] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:4021[RHBA-2025:4021] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.18.10 --pullspecs
+----
+
+[id="ocp-4-18-10-enhancements_{context}"]
+==== Enhancements
+
+*  In the bootstrap phase of the installation process, the transport layer security (TLS) between the `metal3` `httpd` server and the node's Baseboard Management Controller (BMC) is enabled by default in {product-title} 4.18 and later. The `httpd` server is on port 6183 instead of port 6180 when TLS is enabled. Disable the TLS setting by adding 'disableVirtualMediaTLS: true' to the Provisioning custom resource (CR) file that is created on the disk. (link:https://issues.redhat.com/browse/OCPBUGS-39404[OCPBUGS-39404])
+
+
+[id="ocp-4-18-10-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the Prometheus remote-write proxy configuration was not correctly applied to the Prometheus user workload custom resource (CR), which caused communication and data collection problems in the cluster. With this release, the user workload monitoring (UWM) Prometheus configurations, including user workload Prometheus, correctly inherits the proxy settings from the cluster proxy resource. (link:https://issues.redhat.com/browse/OCPBUGS-38655[OCPBUGS-38655])
+
+* Previously, when running Red Hat Enterprise Linux CoreOS (RHCOS) in an active environment, the `rpm-ostree-fix-shadow-mode.service` that used to run caused the `rpm-ostree-fix-shadow-mode.service` to fail. With this release, the `rpm-ostree-fix-shadow-mode.service` does not activate when RHCOS does not run from an installed environment. (link:https://issues.redhat.com/browse/OCPBUGS-41625[OCPBUGS-41625])
```

Comment:
```suggestion
* Previously, when running {op-system-first} in an active environment, the `rpm-ostree-fix-shadow-mode.service` that used to run caused the `rpm-ostree-fix-shadow-mode.service` to fail. With this release, the `rpm-ostree-fix-shadow-mode.service` does not activate when {op-system} does not run from an installed environment. (link:https://issues.redhat.com/browse/OCPBUGS-41625[OCPBUGS-41625])
```
If possible, add the appropriate nouns after `rpm-ostree-fix-shadow-mode.service` and `rpm-ostree-fix-shadow-mode.service` to indicate what those things are.